.gitlab-ci.yml

```diff
@@ -0,0 +1,66 @@
+.docker_job: &docker_job
+  image: docker
+  services:
+    - docker:dind
```

Comment:
https://jpetazzo.github.io/2015/09/03/do-not-use-docker-in-docker-for-ci/

Reply:
Though I don't know if you're using public runners and if so, whether avoiding dind is possible with that

Reply:
Yeah, can't mount the docker socket in this case. I'm not terribly concerned on security here lmao, even in the event of any IAM creds or reddit creds leaking, they're all so scoped that it doesn't matter.